platform/viewer/public/html-templates/script-tag.html

```diff
@@ -31,8 +31,9 @@
     var componentRenderedOrUpdatedCallback = function () {
       console.log('OHIF Viewer rendered/updated');
     };
+    var props = { config: window.config };
     window.OHIFViewer.installViewer(
-      window.config, // See `/config/default.js`
+      props, // See `/config/default.js`
```

Comment:
I think we can keep this passing in an object or a function. I'm not sure the `config` key adds value. Then `installViewer` can set it up appropriately for the App component?

Reply:
Got it. I thought to add the config key would allow more stuff beside config to be passed down as props since the installViewer have this param named as props. I'll update the installViewer props argument to config to be more precise about its usage.